### ISSUE TITLE
Bling Desktop - Activity page is blank

### ISSUE THREAD

**Issue by shoey79** [lurume84/bling-desktop#17]
Bling Desktop v 0.1.13.0
Bling Viewer V 0.1.10.0

Windows OS: Windows 10 x64

Since the 17th Sept, the ability to view previous alerts has been lost. Clicking on the Activity page returns a blank page with no video history.

Uninstall and Re-install of Bling Desktop has not resolved.

The result of this issue means that the cloud downloaded video files has not updated either.

Ini file:

[SyncVideo]
LastUpdate=2019-09-17T11:56:23+00:00
Enabled=true
UseLocalTime=false
Interval=60
Sleep=20
Output=Z:\Tech-Files\CCTV\Videos\
[SyncThumbnail]
Enabled=true
Interval=3600
Sleep=5
Retries=10

This has worked fine since 4th July 2019 when it was first installed.

Not sure if Blink have changed anything that has resulted in this issue?

Thumbnails, Snapshots and live view still work so is just the Activity page.

**Comment by lurume84** [comment]
Solved! I have updated the application with the last API changes and everything should work again. This version has huge changes as uses a new borderless window. Also UI has changed so some issues may appear.

I suggest removing folder Bling.exe\Download\Versions and Bling.exe\Html folders before executing new version to use the new upgrade system.

Hope this works for you!

**Comment by shoey79** [comment]
Thank you very much! 

Yes it works! I removed the folders as advised, the auto upgrade attempted to upgrade and whilst I could see differences, it kept crashing.

I then uninstalled it, removed all the files and downloaded the new exe from the site here, installed that, restored the ini file and it now appears to be working fine, I can see the activity page and the videos download automatically to the folder.

Thank you again for your efforts!
